app/lib/potential_friendship_tracker.rb

```diff
@@ -28,10 +28,14 @@ def remove(account_id, target_account_id)
       redis.zrem("interactions:#{account_id}", target_account_id)
     end
 
-    def get(account_id, limit: 20, offset: 0)
-      account_ids = redis.zrevrange("interactions:#{account_id}", offset, limit)
-      return [] if account_ids.empty?
-      Account.searchable.where(id: account_ids)
+    def get(account, limit: 20, offset: 0)
+      account_ids          = redis.zrevrange("interactions:#{account.id}", offset, limit)
+      fallback_account_ids = redis.zrevrange("follow_recommendations:#{account.user_locale}", 0, -1)
+
+      [].tap do |accounts|
+        accounts.concat(Account.searchable.where(id: account_ids)) unless account_ids.empty?
+        accounts.concat(Account.followable_by(account).not_excluded_by_account(account).not_domain_blocked_by_account(account).where(id: fallback_account_ids).where.not(id: account.id).limit(limit - accounts.size)) if accounts.size < limit && offset.zero? && fallback_account_ids.size.positive?
+      end
```

Comment:
Just noticed two things:
- result isn't sorted by score (that's true for both the old `PotentialFriendshipTracker` results and the new generic suggestions
- results may be duplicated between the personalized set and the generic suggestions

Thinking about it, I think this would probably be better as a separate endpoint, or a new version of the endpoint, so that personalized and generalized suggestions can be differentiated in the UI. This would also make the fact that generic suggestions cannot be ignored less confusing.